docs/tutorials/getting_started.md

```diff
@@ -61,118 +65,232 @@ directory:
 ```
 
-## 3. Deploy SD-Core
+## 3. Install Terraform
 
+From your terminal, install Terraform.
 
-Create a Juju model named `core`:
+```console
+sudo snap install terraform --classic
+```
+
+## 4. Create Terraform module
+
+On the host machine create a new directory called `terraform`:
 
 ```console
-juju add-model core
+mkdir terraform
 ```
 
-Deploy the `sdcore-router-k8s` operator:
+Inside newly created `terraform` directory create a `terraform.tf` file:
 
 ```console
-juju deploy sdcore-router-k8s router --trust --channel=beta
+cd terraform
+cat << EOF > terraform.tf
+terraform {
+  required_providers {
+    juju = {
+      source  = "juju/juju"
+      version = "~> 0.10.1"
+    }
+  }
+}
+EOF
 ```
 
-Deploy the `sdcore-k8s` charm bundle:
+Create a Terraform module containing the SD-Core 5G core network, 5G radio and a cellphone
+simulator and a router:
 
 ```console
-juju deploy sdcore-k8s --trust --channel=beta
+cd terraform
+cat << EOF > main.tf
+resource "juju_model" "sdcore" {
+  name = "sdcore"
+}
+
+module "sdcore-router" {
+  source = "git::https://github.com/canonical/sdcore-router-k8s-operator//terraform"
+
+  model_name = juju_model.sdcore.name
+  depends_on = [juju_model.sdcore]
+}
+
+module "sdcore" {
+  source = "git::https://github.com/canonical/terraform-juju-sdcore-k8s//modules/sdcore-k8s"
+
+  model_name = juju_model.sdcore.name
+  create_model = false
+  
+  traefik_config = {
+    routing_mode = "subdomain"
+  }
+
+  depends_on = [module.sdcore-router]
+}
+
+module "gnbsim" {
+  source = "git::https://github.com/canonical/sdcore-gnbsim-k8s-operator//terraform"
+
+  model_name = juju_model.sdcore.name
+  depends_on = [module.sdcore-router]
+}
+
+resource "juju_integration" "gnbsim-amf" {
+  model = juju_model.sdcore.name
+
+  application {
+    name     = module.gnbsim.app_name
+    endpoint = module.gnbsim.fiveg_n2_endpoint
+  }
+
+  application {
+    name     = module.sdcore.amf_app_name
+    endpoint = module.sdcore.fiveg_n2_endpoint
+  }
+}
+
+resource "juju_integration" "gnbsim-nms" {
+  model = juju_model.sdcore.name
+
+  application {
+    name     = module.gnbsim.app_name
+    endpoint = module.gnbsim.fiveg_gnb_identity_endpoint
+  }
+
+  application {
+    name     = module.sdcore.nms_app_name
+    endpoint = module.sdcore.fiveg_gnb_identity_endpoint
+  }
+}
+EOF
 ```
 
-Deploying the core network can take up to 15 minutes. You can validate the status of the
-deployment by running `juju status`. The core network is ready when all the charms are in the
-`Active/Idle` state. It is normal for `grafana-agent-k8s` to remain in waiting state. Example:
+```{note}
+You can get a ready example by cloning [this Git repository](https://github.com/canonical/charmed-5g).
+All necessary files are in the `examples/terraform/getting_started` directory.
+```
+
+## 5. Deploy SD-Core
+
+Initialize Juju Terraform provider:
+
+```console
+terraform init
+```
+
+Deploy 5G network.
+
+```console
+terraform apply -auto-approve
```

Comment:
I also get `This object does not have an attribute named "database_endpoint".`

Perhaps this is due to the change in auth databases, @gatici ?

Reply:
Yes, that's exactly the reason. I've updated the modules, so it should be good again.